lib/pdk/cli/config/get.rb

```diff
@@ -0,0 +1,24 @@
+module PDK::CLI
+  @config_get_cmd = @config_cmd.define_command do
+    name 'get'
+    usage _('config get [name]')
+    summary _('Retrieve the configuration for <name>. If not specified, retrieve all configuration settings')
+
+    run do |_opts, args, _cmd|
+      item_name = args[0]
+      resolved_config = PDK::Config.new.resolve(item_name)
```

Comment:
```suggestion
      resolved_config = PDK.config.resolve(item_name)
```
There's a memoised accessor for `PDK::Config` that you can use instead of instantiating a new one

Reply:
Good catch! fixed.